drivers/gles3/storage/particles_storage.cpp

```diff
@@ -1120,8 +1120,8 @@ void ParticlesStorage::update_particles() {
 			double delta = RSG::rasterizer->get_frame_delta_time();
 			if (delta > 0.1) { //avoid recursive stalls if fps goes below 10
 				delta = 0.1;
-			} else if (delta <= 0.0) { //unlikely but..
-				delta = 0.001;
+			} else if (delta < 0.0) { //unlikely but..
```

Comment:
You could update the comment if you are modifying the line anyway:
```suggestion
			} else if (delta < 0.0) { // Unlikely, but..
```
Or it can be removed altogether, as it's not very useful.

Reply:
done :)